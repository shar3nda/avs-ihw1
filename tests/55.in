11
-2 0 1 7 1 3 6 4 2 -2 3