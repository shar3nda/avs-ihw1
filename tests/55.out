-1 4 7 10 -3 5 8 -4 3 9